src/System.ValueTuple/src/System.ValueTuple.csproj

```diff
@@ -38,4 +38,8 @@
     </When>
   </Choose>
 
-</Project>
\ No newline at end of file
+  <Target Name="FailForNet47AsNetMinimum" Condition="'$(NetFrameworkMinimum)' == 'net47'">
```

Comment:
This target never runs as it doesn't have a hook point.

Reply:
Project InitialTargets